src/messageFilters.ts

```diff
@@ -0,0 +1,18 @@
+import { Message } from 'discord.js';
+
+export type MessageFilter = (message: Message) => boolean;
+
+export async function runMessageFilters(message: Message, messageFilters: MessageFilter[]): Promise<void> {
```

Comment:
I would suggest just taking a single filter here and removing the looping logic.

Reply:
Why?

Reply:
It pulls the looping responsibility outside of the function which I think is slightly better. The function doesn't really need to know whether there's multiple filters or not. I guess the justification imo is similar to why you wouldn't take `messages: Message[]`. Whether or not there's multiple doesn't really matter for the functionality (other than maybe this sequential execution thing below I need to read about).